### PR TITLE
Adding Code of Conduct WG charter

### PR DESCRIPTION
**Pull request by joesepi**
Work done in today's meeting

### CODE REVIEW

File: WORKING_GROUPS.md [hunk 1]
```diff
@@ -129,8 +129,24 @@ The [OpenJS Code of Conduct][] applies to this Working Group.
...
 * [Standards](#standards)
 
+### [Code of Conduct](https://github.com/openjs-foundation/code-of-conduct/)
+
+The purpose of the Code of Conduct Working Group is to define and maintain a set of resources/processes for use by the Foundation, its teams and its projects.
+
+Responsibilities include:
+
+* Define and maintain overarching Code of Conduct and moderation processes, documentation and resources as they pertain to the Foundation and the Cross Project Council.
+* Define minimum Code of Conduct and moderation requirements for projects as part of Foundation membership and provide guidance/resources to help them implement those requirements.
```
Comment (mcollina) [inline WORKING_GROUPS.md:142]: I do not expect the mininum requirements to be changed without an approval with the CPC. I think it is important that https://github.com/openjs-foundation/cross-project-council/blob/master/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md stays under the purview of the CPC and its reponsibility not to be transfered.

In other terms, I do not see what this working group could add to that document? If we think that should be changed/integrated, I think it should be proposed by the wg and approved by the CPC.

I recommend this to be changed to:

```suggestion
* Provide guidance/resources to implement the minimum Code of Conduct and moderation requirements for projects as part of Foundation membership as defined in https://github.com/openjs-foundation/cross-project-council/blob/master/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md.
```

File: WORKING_GROUPS.md [hunk 2]
```diff
@@ -129,8 +129,24 @@ The [OpenJS Code of Conduct][] applies to this Working Group.
...
+* Define and maintain template Code of Conduct and moderation processes and documentation as resources for the Foundation’s projects to use in whole or as guidance to develop their own processes, documentation and resources, including training materials
```
Comment (ljharb) [inline WORKING_GROUPS.md:143]:
```suggestion
* Define and maintain template Code of Conduct and moderation processes and documentation as resources for the Foundation’s projects to use in whole or as guidance to develop their own processes, documentation, and resources, including training materials
```
oops, missed this one earlier

**Comment by jorydotcom** [comment]
Should this also be PR'd into the code of conduct repo?

**Comment by mhdawson** [comment]
@jorydotcom I think at least  a link to the charter in the repo makes sense. I think there is often a copy but that means double maintenance.

**Comment by jorydotcom** [comment]
@mhdawson okiedok - for some reason I thought the standards wg had their charter in their repo but I was wrong

File: WORKING_GROUPS.md [hunk 3]
```diff
@@ -129,8 +129,24 @@ The [OpenJS Code of Conduct][] applies to this Working Group.
...
 * [Standards](#standards)
 
+### [Code of Conduct](https://github.com/openjs-foundation/code-of-conduct/)
+
+The purpose of the Code of Conduct Working Group is to define and maintain a set of resources/processes for use by the Foundation, its teams and its projects.
+
+Responsibilities include:
+
+* Define and maintain overarching Code of Conduct and moderation processes, documentation and resources as they pertain to the Foundation and the Cross Project Council.
+* Provide guidance/resources to implement the minimum Code of Conduct and moderation requirements for projects as part of Foundation membership as defined in https://github.com/openjs-foundation/cross-project-council/blob/master/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md.
```
Comment (sendilkumarn) [inline WORKING_GROUPS.md:142]:
```suggestion
* Provide guidance/resources to implement the minimum Code of Conduct and moderation requirements for projects as part of Foundation membership as defined [here](https://github.com/openjs-foundation/cross-project-council/blob/master/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md).
```

Comment (timmywil) [reply WORKING_GROUPS.md:142]: Perhaps instead of "here" it could say `defined in the [Foundation Code of Conduct Requirements]...`

Comment (joesepi) [reply WORKING_GROUPS.md]: Updated. Thanks!

**Comment by mhdawson** [comment]
No objections, greater than 2 approvals and has been open for more than 14 days.  Believe it is ready to land.

**Pull request merged**